flaskApi/app_service.py

```diff
@@ -0,0 +1,49 @@
+import parser
+import file_db
+import util
+db = file_db.Filedb()
+
+
+def upload_pptx(pptx_file):
```

Comment:
Separate code to small functions.  Read more: https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be